src/http-driver.js

```diff
@@ -103,14 +108,12 @@ function makeHTTPDriver({eager = false} = {eager: false}) {
       .map(reqOptions => {
         let response$ = createResponse$(reqOptions)
         if (eager || reqOptions.eager) {
-          response$ = response$.replay(null, 1)
-          response$.connect()
+          response$ = response$.publishReplay(null, 1).connect()
+          return response$
         }
         response$.request = reqOptions
         return response$
-      })
-      .replay(null, 1)
-    response$$.connect()
+      }).shareReplay(null, 1)
```

Comment:
shareReplay is replay().refCount(), which is different than replay() + connect(). I'd advise to keep the replay() + connect().